static/ejs/partials/scripts/utils.ejs

```diff
@@ -9,7 +9,19 @@
   const purpleAiHtmlETL = <%- purpleAi.htmlETL %>;
   const purpleAiRules = <%- JSON.stringify(purpleAi.rules) %>;
 
-  const htmlTagAndAttributeRegex = new RegExp(/([a-zA-Z][^\s/>]*\b)|([\w-]+)\s*=\s*["']([^"']*)["']/g);
+  const getHtmlTagAndAttributes = (htmlString) => {
```

Comment:
Could you please give an example or comment on whats happening here🙏 . A little hard to follow